src/css/App.css

```diff
@@ -1,14 +1,14 @@
 @import url('https://fonts.googleapis.com/css?family=Lato:400,700&display=swap');
 
-/* Globals */
+/* Globals Steven Ribeiro*/
```

Comment:
Please avoid leaving a signature in the codem the PR will be attributed to you

Reply:
ok thanks